tests/util/ginkgo/lib.go

```diff
@@ -389,28 +390,32 @@ func (ns *NsWrapper) WaitForDatacenterReadyPodCountWithTimeout(dcName string, co
 	step := "waiting for the node to become ready"
 	json := "jsonpath={.items[*].status.containerStatuses[0].ready}"
 	k := kubectl.Get("pods").
-		WithLabel(fmt.Sprintf("cassandra.datastax.com/datacenter=%s", dcName)).
+		WithLabel(fmt.Sprintf("cassandra.datastax.com/datacenter=%s", api.CleanupForKubernetes(dcName))).
 		WithFlag("field-selector", "status.phase=Running").
 		FormatOutput(json)
 	ns.WaitForOutputAndLog(step, k, duplicate("true", count), timeout)
 }
 
 func (ns *NsWrapper) WaitForDatacenterReady(dcName string) {
-	ns.WaitForDatacenterReadyWithTimeouts(dcName, 1200, 1200)
+	ns.WaitForDatacenterReadyWithTimeouts(dcName, dcName, 1200, 1200)
+}
+
+func (ns *NsWrapper) WaitForDatacenterReadyWithOverride(dcName, dcNameOverride string) {
+	ns.WaitForDatacenterReadyWithTimeouts(dcName, dcNameOverride, 1200, 1200)
 }
 
 func (ns *NsWrapper) Log(step string) {
 	ginkgo.By(step)
 }
 
-func (ns *NsWrapper) WaitForDatacenterReadyWithTimeouts(dcName string, podCountTimeout int, dcReadyTimeout int) {
+func (ns *NsWrapper) WaitForDatacenterReadyWithTimeouts(dcName string, dcNameOverride string, podCountTimeout int, dcReadyTimeout int) {
```

Comment:
Instead of replicating that dcNameOverride parameter everywhere, wouldn't it have made more sense to get the overrideName from that object we just fetched?

Reply:
Of course!

brilliant idea 👍